src/imagery/i.eodag/i.eodag.py

```diff
@@ -300,82 +400,199 @@ def no_fallback_search(search_parameters, provider):
     search_result = dag.search_iter_page(**search_parameters)
 
     # TODO: Would it be useful if user could iterate through
-    # the pages manually, and look for the product themselves?
+    #       the pages manually, and look for the product themselves?
     try:
         # Merging the pages into one list with all products
-        return [j for i in search_result for j in i]
+        return SearchResult([j for i in search_result for j in i])
     except Exception as e:
-        gs.verbose(e)
+        gs.debug(e)
         gs.fatal(_("Server error, please try again."))
 
 
-def create_products_dataframe(eo_products):
-    result_dict = {"id": [], "time": [], "cloudCover": [], "productType": []}
-    for product in eo_products:
-        for key in result_dict:
-            if key == "time":
-                if (
-                    "startTimeFromAscendingNode" in product.properties
-                    and product.properties["startTimeFromAscendingNode"] is not None
-                ):
+def list_products(products):
+    """Lists products on the Standard Output (Console) stream.
+
+    :param products: EO poducts to be listed
+    :type products: class:'eodag.api.search_result.SearchResult'
+    """
+    columns = ["id", "publicationDate", "cloudCover", "productType"]
```

Comment:
@HamedElgizery do you know what publication date actually means? I had a look around and didn't find the definition. To me it sounds like the date when the product was put online, I think ingestion date is more relevant, i.e. when the sensor recorded the data. But I didn't find ingestionDate, I only found creationDate... whatever that means.

Reply:
Yeah right, just checked...

Apparently:
* `publicationDate` is the date they was put online.
* `startTimeFromAscendingNode` is the time when the scene was recorded.

Fixed here [6d99e7f](https://github.com/OSGeo/grass-addons/pull/1104/commits/6d99e7fc3f5b43adab54a8ae432b29e1ec623719).